### PR TITLE
Tidied up

### PR DESCRIPTION
**Pull request by JonahSussman**
- Removed globals
- Renamed kaffine-mod to github.com/konveyor/kaffine
- Moved contents of tool/ to root repo directory
- Removed SHA1 helper function
- Implemented rudimentary file saving safety (.bak)

### CODE REVIEW

File: cmd/config/config.go [hunk 1]
```diff
@@ -3,51 +3,63 @@ package config
...
+	"github.com/konveyor/kaffeine/kaffeine"
 
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 )
 
 // support both local and global config
-// global config determined by KAFFINE_GLOBAL_CONFIG env variable.
-// If unset, defaults to ~/.kaffine/config
+// global config determined by kaffeine_GLOBAL_CONFIG env variable.
```
Comment (shawn-hurley) [inline cmd/config/config.go]: nit I think you still want it to be capitalized

File: kaffeine/catalog_manager.go [hunk 2]
```diff
@@ -163,37 +192,40 @@ func (cm *CatalogManager) RemoveCatalog(uri string) (oldFc FunctionCatalog, err
...
 func (cm *CatalogManager) UpdateCatalog(uri string) (oldFc FunctionCatalog, err error) {
-	// fmt.Println("UPDATING", uri)
-	// fmt.Println("REMOVING", uri)
 	oldFc, err = cm.RemoveCatalog(uri)
 	if err != nil {
 		return
 	}
 
-	// fmt.Println("GETTING EXTERNAL", uri)
+	revertToOldFc := func() {
```
Comment (shawn-hurley) [inline kaffeine/catalog_manager.go]: not usually a huge fan of closures inside a function. 

I would do potentially two things
1. make this it's own function or
2. Potentially add it in a recover block, so if something panics, it recovers the backup, https://gobyexample.com/recover

File: kaffeine/catalog_manager.go [hunk 3]
```diff
@@ -261,38 +287,39 @@ func (cm *CatalogManager) Search(fname string, lowercase bool) (fns []FunctionDe
...
+// 		fn.Versions = versions
+// 	}
+
+// 	return
+// }
+
+// // UNUSED
+// func (cm *CatalogManager) SearchMultiple(fnames []string) (fnss [][]FunctionDefinition, errs []error) {
+// 	return
+// }
```
Comment (shawn-hurley) [inline kaffeine/catalog_manager.go]: lets just remove unused code for now

**Review by shawn-hurley — CHANGES_REQUESTED** [review]
I think there is a convention in golang for the godocs to start with the function name or the struct name or the field name. Can we update for that as well?

File: cmd/config/config.go (hunk 1 above)
Comment (JonahSussman) [reply cmd/config/config.go]: This was definitely a find-and-replace error, whoops!

**Review by shawn-hurley — APPROVED** [review]
LGTM

**Pull request merged**